### PR TITLE
Adding numThreads as a Jester setting

### PR DESCRIPTION
**Pull request by jasonprogrammer**
This PR assumes that a new release of httpbeast has been created with the numThreads changes recently merged to master

### CODE REVIEW

File: jester.nim [hunk 1]
```diff
@@ -494,7 +496,8 @@ proc serve*(
       proc (req: httpbeast.Request): Future[void] =
          {.gcsafe.}:
           result = handleRequest(jes, req),
-      httpbeast.initSettings(self.settings.port, self.settings.bindAddr)
+      httpbeast.initSettings(self.settings.port, self.settings.bindAddr,
+                             self.settings.numThreads)
```
Comment (dom96) [inline jester.nim:500]: Maybe you could support both by using `when compiles(...)` here?

Comment (jasonprogrammer) [reply jester.nim:500]: Sure, I can add that. Will a new version of httpbeast be released so I can reference it in the PR?

**Comment by dom96** [comment]
Will we get a new PR?

**Comment by jasonprogrammer** [comment]
Yes, I'd like to create a new PR. These changes depend on https://github.com/dom96/httpbeast/commit/3903fa591776f2e6ab0fee348ee2d08290819a89, and I'm not sure what version of httpbeast to refer to as a part of the PR

**Pull request closed without merging**